src/modules/orbitdb/indexes/sql.js

```diff
@@ -196,6 +212,7 @@ class ChluSQLIndex extends ChluAbstractIndex {
         let entity = await this._readDID(publicDidDocument.id)
         if (!entity) {
             entity = await this.DID.create({
+                chluNetwork: this.chluIpfs.network,
```

Comment:
Should we track chlu version here too? @fazo96 

And referring to earlier discussion, I am happy to go with the extra field approach you were suggesting, instead of bitmaps :)

Reply:
Nope, I don't think it's needed to track chlu version here.

I will link to this from the review network issue https://github.com/ChluNetwork/chlu-ipfs-support/issues/100